### DISCUSSION TITLE
Steam API raise error 403: Please use V2 filtered and sample volume stream as alternatives

### DISCUSSION THREAD

**Discussion by viokuma** [tweepy/tweepy#2081] [Questions] [answered]
Today I running many years code raise 403 error.
It using stream api:

`myStreamListener = tweepy.Stream(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
    myStreamListener.filter(follow=['test'])`

error:
<img width="1469" alt="image" src="https://user-images.githubusercontent.com/1708895/225256133-956eada3-cf8f-474c-8feb-369d12359e32.png">

Looks like V1.1 is disabled.But V2 needs verify user info.
How to fix it?

**Comment by Harmon758** [comment, marked as answer]
[The Twitter API v1.1 statuses/filter endpoint was deprecated on the 9th.](https://twittercommunity.com/t/announcing-the-deprecation-of-v1-1-statuses-filter-endpoint/182960)
This seems to have taken effect yesterday.

[`Stream` (and `AsyncStream`) was deprecated with Tweepy v4.13.0](https://github.com/tweepy/tweepy/releases/tag/v4.13.0) and will be removed in v4.14.0.

As the error indicates, you'll have to use Twitter API v2 instead.

**Comment by okadato623** [comment]


**Reply by Harmon758** [reply]
That's not relevant at all to this discussion.
Please read your error.
